app/jupyterlite.schema.v0.json

```diff
@@ -86,12 +86,6 @@
           "default": "./extensions",
           "$ref": "#/definitions/non-trailing-slash-uri"
         },
-        "fullMathjaxUrl": {
```

Comment:
i guess removing this doesn't _strictly_ require a version number bump, as `additionalProperties: true`....